{"uuid":"ca765f4e-f4be-4f62-a56e-a3ede8dd4174","historyId":"2c2ad341d427602da7cfb04a224c4771","fullName":"org.thetestingmap.testingchallenges.challenge_1.NegativeFirstNameTest.negativeLogInTest","labels":[{"name":"package","value":"org.thetestingmap.testingchallenges.challenge_1.NegativeFirstNameTest"},{"name":"testClass","value":"org.thetestingmap.testingchallenges.challenge_1.NegativeFirstNameTest"},{"name":"testMethod","value":"negativeLogInTest"},{"name":"parentSuite","value":"suite"},{"name":"suite","value":"firstName-firefox"},{"name":"subSuite","value":"org.thetestingmap.testingchallenges.challenge_1.NegativeFirstNameTest"},{"name":"host","value":"DESKTOP-8H0MF42"},{"name":"thread","value":"16496@DESKTOP-8H0MF42.TestNG-tests-2(14)"}],"links":[],"name":"negativeLogInTest","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.RuntimeException: File src\\test\\resources\\dataproviders\\NegativeFirstNameTest\\negativeLogInTest.csv was not found.\n[Ljava.lang.StackTraceElement;@3be03594","trace":"java.lang.RuntimeException: java.lang.RuntimeException: File src\\test\\resources\\dataproviders\\NegativeFirstNameTest\\negativeLogInTest.csv was not found.\n[Ljava.lang.StackTraceElement;@3be03594\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:50)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:146)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:820)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:762)\r\n\tat org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:60)\r\n\tat org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:39)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:771)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.util.ArrayList.forEach(ArrayList.java:1257)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:770)\r\n\tat org.testng.TestRunner.run(TestRunner.java:591)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:402)\r\n\tat org.testng.SuiteRunner.access$000(SuiteRunner.java:41)\r\n\tat org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:443)\r\n\tat org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)\r\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:266)\r\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)\r\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)\r\n\tat java.lang.Thread.run(Thread.java:748)\r\nCaused by: java.lang.RuntimeException: File src\\test\\resources\\dataproviders\\NegativeFirstNameTest\\negativeLogInTest.csv was not found.\n[Ljava.lang.StackTraceElement;@3be03594\r\n\tat org.thetestingmap.testingchallenges.testEngine.CsvDataProviders.csvReader(CsvDataProviders.java:40)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.lang.reflect.Method.invoke(Method.java:498)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:77)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:46)\r\n\t... 20 more\r\n"},"stage":"finished","start":1592045048000,"stop":1592045048006,"parameters":[{"name":"browser","value":"firefox"}]}